install_process_test.go

```diff
@@ -189,15 +183,12 @@ func testInstallProcess(t *testing.T, context spec.G, it spec.S) {
 				"--modules-folder",
 				filepath.Join(modulesLayerPath, "node_modules"),
 			}))
-			Expect(executions[1].Env).To(ContainElement(MatchRegexp(`^PATH=.*:node_modules/.bin$`)))
 
```

Comment:
```suggestion
Expect(executions[1].Env).To(ContainElement(MatchRegexp(`^PATH=.*:node_modules/.bin$`))
```
Isn't this still relevant?